requirements/pip.in

```diff
@@ -1,3 +1,3 @@
-pip<21.0  # Our hack for installing specific commits from Git requires --use-deprecated=legacy-resolver: https://github.com/pypa/pip/issues/5780
+https://github.com/andersk/pip/archive/6642149c44297200f44dfe215b1cb7954d47ed77.zip#egg=pip==20.3.4+git  # Our hack for installing specific commits from Git requires --use-deprecated=legacy-resolver: https://github.com/pypa/pip/issues/5780
```

Comment:
Do we want to move this fork into the `zulip` organization before shipping link with this URL?